### PR TITLE
ensure core dumps created in fat tests that go to /var are available …

### LINKED ISSUE #28976: Fat test core dumps are getting lost
We are seeing fat tests where the logs say a core dump was produced but there is nothing uploaded in the zip files of logs and server details. This may be because they are being sent to /var/crash/ or /var/lib/apport/coredump/

The ant.xml file that launches the fat tests already has code to put the dumps where they are needed. It could be patched to include those two folders. 

See this slack discussion: https://ibm-cloud.slack.com/archives/C59HR9D5X/p1719988353798819?thread_ts=1719941528.668509&cid=C59HR9D5X

### PR DESCRIPTION
**Pull request by benjamin-confino**
…for debugging

fixes #28976

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

File: dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml [hunk 1]
```diff
@@ -691,6 +691,25 @@
...
+		<fileset id="javacores.test" dir="/var/crash/">
+			<include name="javacore*" />
+			<include name="core*" />
+			<include name="heapdump*.phd" />
+			<include name="Snap*" />
+			<include name="*.hprof" />
+			<include name="hs_err*" />
+		</fileset>
+
+		<fileset id="javacores.test" dir="/var/lib/apport/coredump/">
```
Comment (websterc87) [inline dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml]: You will need to make different refids and then add them to the move down below

File: dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml [hunk 2]
```diff
@@ -691,6 +691,25 @@
 			<exclude name="${javacores.server.exclude}" />
 		</fileset>
 
+		<!-- Move any test javacores into the javacores directory that were piped to /var, there are two places to check -->
+		<fileset id="javacores.test" dir="/var/crash/">
```
Comment (websterc87) [inline dev/fattest.simplicity/autoFVT-defaults/src/ant/launch.xml]: You will need to make different refids and then add them to the move down below approx line 702

**Review by websterc87 — CHANGES_REQUESTED** [review]
Overrides the existing move of core dumps we need this to be additional.

**Review by websterc87 — DISMISSED** [review]
Happy assuming it doesnt break during test due to those directories not existing.

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
Sadly the assumption was not true, after testing, I rewrote with uglier code that checks for those directories explicitly.

**Review by websterc87 — CHANGES_REQUESTED** [review]
total.length.of.all.javacores is being overloaded and will no longer be accurate

**Review by websterc87 — APPROVED** [review]

**Comment by benjamin-confino** [comment]
Three test failures. None of them look related to core dump files, but one was related to filepaths. This directory was not found `extractDir:/home/jazz_build/wlpExtract/runnableTestServer_25914883716460/wlp`

I think its unrelated but to be sure I'll run a second build and see if it happens again. 

The other two were an assertion failure in BatchPartitionOldDBTablesTest.testRemotablePartitionOlderTables and an java.lang.IllegalStateException: Bundle has been uninstalled

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
Only one consistent failure across the builds and it matches a pre-existing defect: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.workitem.viewWorkItem&id=295756

**Comment by benjamin-confino** [comment]
#libby

**Pull request merged**